docs/development/build-instructions-windows.md

```diff
@@ -5,8 +5,15 @@ Follow the guidelines below for building Electron on Windows.
 ## Prerequisites
 
 * Windows 10 / Server 2012 R2 or higher
-* Visual Studio 2017 15.7.2 or higher - [download VS 2017 Community Edition for
+* Visual Studio 2017 15.7.2 or higher - [download VS 2019 Community Edition for
   free](https://www.visualstudio.com/vs/)
+  * Ensure that the "Desktop development with C++" workload is included in your installation,
```

Comment:
We should link to https://chromium.googlesource.com/chromium/src/+/master/docs/windows_build_instructions.md#Visual-Studio and/or include the list of needed dependencies listed there

Reply:
Done. Thanks for pointing that out -- that doc is really useful!